index.js

```diff
@@ -12,6 +10,7 @@ var ECMA_SIZES  = require('./byte_size');
  * @returns {*}
  */
 function sizeof(object) {
+  try {
     if (_.isObject(object)) {
```

Comment:
This function does not count  internal hasmap size for the objects. It may assume at least 8 bytes to store reference to any object. And also, each object have fixed header size. So there are many ways how to improve this function